Solution1/BusinessLayer/Service/WeatherService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using AutoMapper;
+using BusinessLayer.Abstract;
+using BusinessLayer.DTOs;
+using BusinessLayer.Service.Abstract;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace BusinessLayer.Service
+{
+    public class WeatherService : IWeatherServise
+    {
+        readonly IMapper _mapper;
```

Comment:
it's good style to specify access modifiers explicitly

Reply:
+